src/valkey-benchmark.c

```diff
@@ -1552,7 +1558,11 @@ int parseOptions(int argc, char **argv) {
         "                    on the command line.\n"
         " -I                 Idle mode. Just open N idle connections and wait.\n"
         " -x                 Read last argument from STDIN.\n"
-        " --seed <num>       Set the seed for random number generator. Default seed is based on time.\n",
+        " --seed <num>       Set the seed for random number generator. Default seed is based on time.\n"
+        " --num-functions <num>     Sets the number of functions present in the Lua lib that is\n"
+        "                           loaded when running the 'function_load' test. (default 10).\n"
+        " --num-keys-in-fcall <num> Sets the number of keys passed to FCALL command when running\n"
+        "                           the 'fcall' test. (default 1)\n",
```

Comment:
In man pages, option lists are rendered with a line break when the option doesn't fit to the left of the aligned descriptions, similar to this:

```suggestion
        " --seed <num>       Set the seed for random number generator. Default seed is based on time.\n"
        " --num-functions <num>\n"
        "                    Sets the number of functions present in the Lua lib that is\n"
        "                    loaded when running the 'function_load' test. (default 10).\n"
        " --num-keys-in-fcall <num>\n"
        "                    Sets the number of keys passed to FCALL command when running\n"
        "                    the 'fcall' test. (default 1)\n",
```

I think it looks better. WDYT?